keylime/tenant.py

```diff
@@ -825,17 +865,20 @@ def do_cvdelete(self, verifier_check: bool = True) -> None:
 
             if deleted:
                 logger.info(
-                    "Agent ID %s deleted from the Verifier after %d tries",
-                    self.agent_uuid,
+                    "%s was deleted from %s after %d tries",
+                    self.agent_fid_str,
+                    self.verifier_fid_str,
                     numtries,
                 )
-
+                # Marked for deletion (need to modify the code on CI tests)
```

Comment:
Any alternative then?

Reply:
Good question... Here is my plan (@kkaarreell and @Koncpa please provide input if you see an alternative): 
1) re-add all the "ugly" messages which will match the regular expressions required by the tests, make sure everything passes and merge. 
2) open a PR on https://github.com/RedHat-SP-Security/keylime-tests.git (with the new regular expressions), make sure it passes and merge
3) open a new PR here, and eliminate all legacy messages

Reply:
Yes, if possible, please do the required tests updates and temporary redirect Packit CI plan in this PR to your branch. This way you can test those changed and later we merge both.

Reply:
> Yes, if possible, please do the required tests updates and temporary redirect Packit CI plan in this PR to your branch. This way you can test those changed and later we merge both.

In order to do that point `url` and `ref` in `packit-ci.fmf` starting lines `44` and `105` to your keylime-tests fork and branch.